chapter03/src/test/java/com/github/sebin/chapter03/BankConfig.java

```diff
@@ -0,0 +1,40 @@
+package com.github.sebin.chapter03;
+import java.util.Calendar;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.github.sebin.chapter03.Account;
+import com.github.sebin.chapter03.service.BankService;
+import com.github.sebin.chapter03.service.impl.XBank;
+
+@Configuration
+public class BankConfig {
+
+	@Bean
+	public User user() {
+		return new User(1, "xbxlnv5", "123456", "My Name");
+	}
+
+
+	@Bean
+	public Account fromAccount() {
+		return new Account(1, user(), "Testing account 1", 10, 100000);
```

Comment:
I am quoting from book
`
Wiring in another bean is a simple matter of referring to that bean’s method. But don’t let the simplicity fool you. More is going on here than meets the eye.

In Spring Java Configuration, referring to a bean through its declaration method isn’t the same as calling the method. If it were, then each time we call sonnet29(), we’d get a new instance of that bean. Spring is more clever than that.
` 
Hope this clarifies.